main.go

```diff
@@ -251,7 +251,7 @@ func message(session *discordgo.Session, e *discordgo.Message) {
 		currentTime := time.Now().In(loc).Format(format)
```

Comment:
Split this variable into `currentTime` and `currentTimeFormat` to avoid function reuse.